src/clib/initialize_chemistry_data.c

```diff
@@ -59,11 +59,108 @@ static void show_version(FILE *fp)
   fprintf (fp, "\n");
 }
 
+/**
+ * Initializes an empty #chemistry_data_storage struct with zeros and NULLs.
+ */
+void init_empty_chemistry_data_storage_struct(chemistry_data_storage *my_rates)
```

Comment:
Just for consistency:
```suggestion
void initialize_empty_chemistry_data_storage_struct(chemistry_data_storage *my_rates)
```

Reply:
cleaned.